client/src/components/Home.js

```diff
@@ -5,7 +5,7 @@ const Home = () => {
   return (
     <div>
       <header className="home-header">
-        <h1>Graduation Project skeleton</h1>
+        <h1>Min-Imece Graduation Project</h1>
```

Comment:
Thank you for updating the title as well! 👍

Reply:
This will cause a conflict with the "Update title" story (which is already merged) so should be removed :slightly_smiling_face: .

Reply:
Yes let's remove this change so that this pull request is focussed on only updating the About page 🙂